DZSwiftUI/DZSwiftUI/Extension/View+Extension.swift

```diff
@@ -13,7 +13,80 @@ extension View {
         modifier(NeumorphismSelected())
     }
 
-    func neumorphismSelectedCircleStyle() -> some View {
+    func neumorphismUnSelectedCircleStyle() -> some View {
         modifier(NeumorphismUnSelectedCircle())
     }
+
+    func neumorphismSelectedCircleStyle() -> some View {
+        modifier(NeumorphismSelectedCircle())
+    }
+
+    var systemsButtonStrokeGradient: LinearGradient {
+        LinearGradient(
+            colors: [Color("topGradient"), Color("bottomGradient")],
```

Comment:
```suggestion
            colors: [Color(""), Color("")],
```
текстовки в константы